src/aegis_ai/agents.py

```diff
@@ -42,15 +40,20 @@ def create_aegis_agent(**kwargs: Any) -> Agent:
 
 rh_feature_agent = create_aegis_agent(
     name="RHFeatureAgent",
-    retries=2,  # FIXME: this should be made configurable, was included as brutish technique for revalidations
-    tools=[osidb_tool, cwe_tool],
+    retries=5,  # FIXME: this should be made configurable, was included as brutish technique for revalidations
+    toolsets=[redhat_cve_toolset],
```

Comment:
This seems to break all evals:
```
pydantic_ai.exceptions.UserError: FunctionToolset defines a tool whose name conflicts with existing tool from the agent: 'osidb_tool'. Rename the tool or wrap the toolset in a `PrefixedToolset` to avoid name conflicts.
```